index.js

```diff
@@ -10,6 +10,8 @@ var DEFAULT_LANGUAGE = "fi";
 
 var replaceableValues = {
 	LABEL_NAME: { default: "{name}" },
+	SOURCES_URL: { default: "api.digitransit.fi/map/v1/" },
```

Comment:
They're used in the function getReplacements(options), rows 94-95
The default value is used to find which string to replace in the base style